utils/commits-helpers.js

```diff
@@ -0,0 +1,48 @@
+const fetch = require('node-fetch')
+
+const { GITHUB_TOKEN } = require('../constants')
+
+// Formulate a list of all commits based on a certain url endpoint
+// for a release tag on Electron
+async function getAll(urlEndpoint) {
+  const objects = []
+  for await (const obj of getAllGenerator(urlEndpoint)) objects.push(obj)
+  return objects
+}
+
+// Generate and iterate through the JSON blob representing commits
+async function* getAllGenerator(urlEndpoint) {
+  let next = urlEndpoint
+  while (true) {
+    const resp = await fetch(next, {
+      headers: { Authorization: `token ${GITHUB_TOKEN}` }
+    })
+
+    console.log(resp)
+
+    if (!resp.ok) {
+      if (resp.headers.get('x-ratelimit-remaining') === '0') {
+        const resetTime = Math.round(resp.headers.get('x-ratelimit-reset') - Date.now() / 1000)
```

Comment:
I think we need to use `Math.ceil()` here to avoid resetting a second too soon

```suggestion
        const resetTime = Math.ceil(resp.headers.get('x-ratelimit-reset') - Date.now() / 1000)
```